includes/class-wc-data-store.php

```diff
@@ -53,7 +53,8 @@ class WC_Data_Store {
 		'product-variation'     => 'WC_Product_Variation_Data_Store_CPT',
 		'shipping-zone'         => 'WC_Shipping_Zone_Data_Store',
 		'webhook'               => 'WC_Webhook_Data_Store',
-		'revenue-report'        => 'WC_Reports_Orders_Data_Store',
+		'report-revenue-stats'  => 'WC_Reports_Orders_Data_Store',
```

Comment:
This not suppose to be `WC_Reports_Revenue_Query`?

Reply:
The query isn't a data store, though . . . WC_Reports_Orders_Data_Store is the data store you would want if you were working with revenue stats to update/query/etc. them.